central/delegatedregistryconfig/datastore/telemetry.go

```diff
@@ -0,0 +1,44 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+// Gather the enabled for config and number of delegated registries.
+func Gather(ds DataStore) phonehome.GatherFunc {
+	return func(ctx context.Context) (map[string]any, error) {
+		ctx = sac.WithGlobalAccessScopeChecker(ctx,
+			sac.AllowFixedScopes(
+				sac.AccessModeScopeKeys(storage.Access_READ_ACCESS),
+				sac.ResourceScopeKeys(resources.Administration),
+			),
+		)
+		cfg, ok, err := ds.GetConfig(ctx)
+		if !ok || err != nil {
```

Comment:
I know this was just merged, but I just want to confirm we want to return an error even when `err == nil` just because `ok` is `false`? @stehessel

Reply:
Nice catch Ross, I missed this - the default value (delegated disabled) should probably be captured.

Reply:
Right thanks, I'll create a fix.

Reply:
fix is here https://github.com/stackrox/stackrox/pull/14019